applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing.h

```diff
@@ -463,13 +468,16 @@ class MapperVertexMorphing : public Mapper
     void InitializeMappingVariables()
     {
         const unsigned int origin_node_number = mrOriginModelPart.Nodes().size();
-        const unsigned int destination_node_number = mrDestinationModelPart.Nodes().size();
+        mValuesOrigin.resize(3);
+        mValuesOrigin[0] = ZeroVector(origin_node_number);
+        mValuesOrigin[1] = ZeroVector(origin_node_number);
+        mValuesOrigin[2] = ZeroVector(origin_node_number);
 
-        mMappingMatrix.resize(destination_node_number,origin_node_number,false);
```

Comment:
resizing of the mapping matrix seems to be missing now?

Reply:
a copy paste mistake from my branch....

Reply:
done :)